### PR TITLE
Updated for APIv5 and added timeamount

### LINKED ISSUE #2: Could you update it again?
Hello? Do you still develop this one? I think may it is better to update the plugin to the latest sponge api 4.xx and 5.xx.

### PR DESCRIPTION
**Pull request by marvin-bitterlich**
Closes #2

Updated to newest sponge api for 1.10.2

added time amount so you can say 15 Minutes.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -21,7 +21,7 @@ repositories {
 }
 
 dependencies {
-    compile "org.spongepowered:spongeapi:4.0.3"
+    compile "org.spongepowered:spongeapi:5.0.0-20160928.153147-131"
 }
```
Comment (hsyyid) [inline build.gradle:25]: Um. You should just put `5.0.0-SNAPSHOT` so it targets latest build

File: src/main/java/io/github/hsyyid/payday/PayDay.java [hunk 2]
```diff
@@ -76,32 +75,23 @@ public void onGameInit(GameInitializationEvent event)
...
+
+				if (subject.getOption("pay").isPresent()) {
+					BigDecimal pay = new BigDecimal(Double.parseDouble(subject.getOption("pay").get()));
+					player.sendMessage(Text.of(TextColors.GOLD, "[PayDay]: ", TextColors.GRAY, "It's PayDay! Here is your salary of " + pay + " dollars! Enjoy!"));
+					UniqueAccount uniqueAccount = economyService.getOrCreateAccount(player.getUniqueId()).get();
+					uniqueAccount.deposit(economyService.getDefaultCurrency(), pay, Cause.of(NamedCause.owner(this)));
 				}
 			}
-		}).interval(1, Utils.getTimeUnit()).name("PayDay - Pay").submit(Sponge.getPluginManager().getPlugin("io.github.hsyyid.payday").get().getInstance().get());
+		}).interval(Utils.getTimeAmount(), Utils.getTimeUnit()).name("PayDay - Pay").submit(Sponge.getPluginManager().getPlugin("payday").get().getInstance().get());
```
Comment (hsyyid) [inline src/main/java/io/github/hsyyid/payday/PayDay.java]: Can't we make this `.submit(this)`??

**Comment by hsyyid** [comment]
Thanks, if you could address the issues I mentioned, I'll be happy to merge this in

**Comment by marvin-bitterlich** [comment]
@hsyyid Sure will do, you just have to submit them ;)

File: src/main/java/io/github/hsyyid/payday/PayDay.java (hunk 2 above)
Comment (marvin-bitterlich) [reply src/main/java/io/github/hsyyid/payday/PayDay.java]: Inherited that from you. If you know it works I can change it.

File: build.gradle (hunk 1 above)
Comment (marvin-bitterlich) [reply build.gradle:25]: Good point will do!

File: src/main/java/io/github/hsyyid/payday/PayDay.java (hunk 2 above)
Comment (hsyyid) [reply src/main/java/io/github/hsyyid/payday/PayDay.java]: Yeah please do, thanks

**Comment by marvin-bitterlich** [comment]
@hsyyid I fixed your remarks

**Comment by hsyyid** [comment]
@Kostronor  Thanks, looks good

**Comment by ProsperCraft** [comment]
You going to build it so we can use it now? :)

**Pull request merged**